src/utils.ts

```diff
@@ -130,7 +131,7 @@ export function formatOrg(input: string, config: Config): string {
 
 export function capitalize(s: string): string {
   if (s === "") return s;
-  return s[0].toUpperCase() + s.substr(1);
```

Comment:
`substr` is a command that will be deprecated, and is being replaced for `substring`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr